src/content/app/genome-browser/hooks/useBrowserRouting.test.tsx

```diff
@@ -105,23 +124,26 @@ const mockState = {
 };
 
 let routingHandle: ReturnType<typeof useBrowserRouting> | null = null;
-const emptyTestContext = {
+const emptyMockRouterContext = {
   url: '',
   pathname: '',
   search: ''
 };
-let testContext: typeof emptyTestContext;
+let mockRouterContext: typeof emptyMockRouterContext;
 
-const TestContext = createContext(emptyTestContext);
+const MockRouterContext = createContext(emptyMockRouterContext);
 
 const TestComponent = () => {
   const { pathname, search } = useLocation();
-  const context = useContext(TestContext);
+  const mockRouterContext = useContext(MockRouterContext);
+
+  useContext(GenomeBrowserIdsContext);
```

Comment:
Where is it being used?

Reply:
Same as the above comment. Forgot to revisit this after fixing the test cases.

Reply:
✅